ucsb-cs156-w24/team03-w24-5pm-3
# Created all Pages for Articles

Implemented all Pages for Articles
Steps for testing:

1. Log in
2. Navigate to the articles index page using NavBar
3. You should be able to see all articles(if any) in the database
4. If an admin, you should be able to see a create, edit, and delete button
5. clicking on create will take you to the create page where you can create a new article, clicking on edit will take you to the edit page where you can edit the article, and clicking on delete will delete that article.

<img width="1440" alt="Screenshot 2024-02-16 at 12 21 50 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-3/assets/79371675/79cb9f13-5849-4e04-889e-d8fb2fe19776">
<img width="1440" alt="Screenshot 2024-02-16 at 12 21 59 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-3/assets/79371675/0db09927-6cc6-4a65-b502-16d6edafc1d8">
<img width="1440" alt="Screenshot 2024-02-16 at 12 22 09 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-3/assets/79371675/30f6ffd4-5134-4ad5-b224-c1e5dfced9ba">
